src/screen3d.ts

```diff
@@ -42,21 +42,21 @@ class Screen3D {
    * Convert the alignment property received from location server into format
    * Babylonjs uses.
    *
-   * - location server "alignment" range: -1 to 0
+   * - location server "alignment" range: -1 to 1.0 TODO fix
```

Comment:
There is bug where the rotation is not done for full turn. Left purposely unfixed as we want to focus on the 2d map now.